lms/static/scripts/frontend_apps/components/test/LMSGrader-test.js

```diff
@@ -84,61 +90,73 @@ describe('LMSGrader', () => {
     );
   });
 
-  it('passes a default value of "0" to onChangeSelectedUser when no a student is selected', () => {
+  it('does not set a focus user by default', async () => {
     renderGrader();
-    assert.isTrue(fakeOnChange.calledWithExactly('0'));
+    await fakeSidebarWindow;
+    assert.isTrue(
+      fakeRpcCall.calledWith(
+        'fake window',
+        'fake origin',
+        'changeFocusModeUser',
+        [
+          {
+            username: undefined,
+            displayName: undefined,
+          },
+        ]
+      )
+    );
   });
-
-  it('passes the unique userid to onChangeSelectedUser when a student is selected', () => {
+  it('calls rpcCall to focus on that user when onSelectStudent is called with a valid student index', async () => {
```

Comment:
Whoa, this is quite a mouthful. Possible to rename more clearly?

Reply:
lol, you should have seen the other version i was going for, it was longer!  Basically this just needs to change, "it changes a user" I 'll make it better